libs/database/src/lib/dtos/common.dto.ts

```diff
@@ -107,3 +114,20 @@ export class CastcleParticipate {
   quoted?: boolean;
   recasted?: boolean;
 }
+
+export class ExpansionQuery {
+  @ApiProperty({
+    enum: UserField,
+    required: false,
+    isArray: true
+  })
+  @TransformStringToArrayOfStrings()
+  @IsOptional()
+  @IsArray()
+  @IsEnum(UserField, { each: true })
+  userFields?: UserField[];
+
+  hasRelationshipExpansion = (() => {
+    return Boolean(this.userFields?.includes(UserField.Relationships));
+  }) as unknown as boolean;
+}
```

Comment:
UserField and ExpansionQuery have already moved to `feed.dto.ts`. I'm not sure that why we need to duplicate them in `common.dto.ts`.